src/main/java/org/folio/rest/impl/OaiPmhViewInstancesAPI.java

```diff
@@ -64,7 +62,7 @@ public void getOaiPmhViewInstances(String startDate, String endDate, boolean del
             respondWithError(completed.cause(), asyncResultHandler);
             return;
           }
-          log.info("Select from oai pmh view completed successfully");
+          log.info(logMessage);
```

Comment:
We don't have much of console output messages and this one is just refactored from former code. So if we need to desrease amount of logging possibly this can be performed over the whole module? How do you think?

Reply:
This is legacy part of code, I just eliminated code duplication. I'm not sure if this particular part of logging is important or not, but it looks it was there before even for production. So how do you think, do we need to change the level of its logging still?

Reply:
yes, please change to `debug` level

Reply:
I changed it to debug level. Thank you for noticing.